superflore/generators/ebuild/overlay_instance.py

```diff
@@ -19,14 +19,14 @@ def get_random_branch_name():
 
 
 class ros_overlay(repo_instance):
```

Comment:
Looking at the initialization. Do you really need the inheritance here or can you just have the repo_instance as a member?